LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2020 Google Inc.
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
I suppose we want to keep Google's copyright, don't we?

Reply:
BTW, since we are touching it we could update it to "Copyright 2023 Google LLC".